examples/release_highlights/plot_release_highlights_0_22_0.py

```diff
@@ -246,9 +246,8 @@ def test_sklearn_compatible_estimator(estimator, check):
 # classification. Two averaging strategies are currently supported: the
 # one-vs-one algorithm computes the average of the pairwise ROC AUC scores, and
 # the one-vs-rest algorithm computes the average of the ROC AUC scores for each
-# class against all other classes. In both cases, the predicted labels are
-# provided in an array with values from 0 to ``n_classes``, and the scores
-# correspond to the probability estimates that a sample belongs to a particular
+# class against all other classes. In both cases, the scores correspond to the
+# probability estimates that a sample belongs to a particular
```

Comment:
I find the expression "the scores" misleading here because just before we mentioned "the ROC AUC scores". I believe that "the scores" in this sentence are the output of `clf.predict_proba(X)`, not the output of `roc_auc_score(y, clf.predict_proba(X), multi_class='ovo')`.

Maybe this could be rephrased as:

"""
In both cases the multiclass ROC AUC scores are computed from the probability estimates that a sample belongs to a particular class according to the model.
"""